tests/checkout/vipps/components/vipps-success.spec.js

```diff
@@ -0,0 +1,17 @@
+import { test, expect } from '@playwright/test';
+
+test('components vipps failure', async ({ page }) => {
```

Comment:
the test title here does not align with the test triggered. Test title is `components vipps failure` and a success flow is triggered.

Suggestion:
`test('components vipps success', async ({ page }) => {...}`

Reply:
I've renamed it to 'success'